### PR TITLE
feat: [#186084688] callout alerts are now aria groups

### PR DESCRIPTION
**Pull request by adidner**
## Description

Bascially we wanted the callout inline alerts to be grouped from the perspective of a screen reader

### Ticket

This pull request resolves [#186084688](https://www.pivotaltracker.com/story/show/186084688).

### Steps to Test

Go to a page with an inline calllout, theres one on step 2 of formation under one of the optional parts below, you do have to open it via the link on the right

image

use a screen reader to navigate through it and see that it has structure relative to the screen reader

### Notes

Made a mapping for content such that they can modify these overrides and text completely. 

Also removed the role from this component it's parent handles this already

## Code author checklist

- [X] I have rebased this branch from the latest main branch
- [X] I have performed a self-review of my code
- [X] I have created and/or updated relevant documentation on the engineering documentation website
- [X] I have not used any relative imports
- [X] I have pruned any instances of unused code
- [X] I have not added any markdown to labels, titles and button text in config
- [X] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [X] I have checked for and removed instances of unused config from CMS
- [X] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts` (see CMS Additions in Engineering Reference/FAQ on the engineering documentation site)
- [X] I have updated relevant `.env` values in both `.env-template` and in Bitwarden

### CODE REVIEW

File: web/src/components/njwds-extended/Alert.tsx [hunk 1]
```diff
@@ -18,12 +20,21 @@ export const AlertVariants = ["info", "success", "warning", "error", "note"] as
...
+    },
+  });
+};
+
 export const Alert = (props: Props): ReactElement => {
   const { variant, children, noIcon, heading, rounded, dataTestid } = props;
   const variantClass = variant ? `usa-alert--${variant}` : "";
   const noIconClass = noIcon ? " usa-alert--no-icon" : "";
   const roundedClass = rounded ? " radius-md" : "";
-  const alertRole = variant === "error" ? { role: "alert" } : {};
```
Comment (adidner) [inline web/src/components/njwds-extended/Alert.tsx:26]: dust - removed this role as when the element is inlined I think announcing it as a group is better and the popup version has separate logic to make sure it's spoken out loud as an alert.

File: web/src/lib/search/cmsCollections.ts [hunk 2]
```diff
@@ -77,6 +77,7 @@ export const cmsCollections = [
       "404 Page",
       "Global Error Messages",
       "Navigation Config",
```
Comment (adidner) [inline web/src/lib/search/cmsCollections.ts:79]: dust - added a new CMS collection for Component Accessibility, this feels like something we need as some components will have specific accessibility content that won't be pairs with any other content.

**Review by FarazA22 — APPROVED** [review]

File: content/src/mappings/callout-alerts.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+  "calloutAlerts": {
+    "calloutAlertGroupTitle": "${alertVariant} alert",
+    "infoOverride": "informational",
+    "successOverride": "success",
+    "errorOverride": "error",
+    "warningOverride": "warning",
+    "noteOverride": "note"
```
Comment (jphechter) [inline content/src/mappings/callout-alerts.json:8]: [sand] Is "override" the right name for these? It feels like this is just being used to set the type, less override a default value.

**Review by jphechter — COMMENTED** [review]
One minor thought in addition to what's already been called out. Otherwise this looks good to me 👍

**Pull request merged**